tests/test_python.py

```diff
@@ -106,13 +107,29 @@ def test_tox(auto_container):
     "container_per_test", CONTAINER_IMAGES_T1, indirect=["container_per_test"]
 )
 @pytest.mark.parametrize("hmodule, port, retry", [("http.server", port1, 10)])
```

Comment:
Why is this parametrized?

Reply:
To improve the parameter visibility and manage possible future change for test reuse or debug.